apps/api/v2/src/modules/slots/services/slots.service.ts

```diff
@@ -40,4 +40,13 @@ export class SlotsService {
 
     return uid;
   }
+
+  async checkIfIsTeamEvent(eventTypeId?: number) {
+    if (!eventTypeId) return false;
+
+    const event = await this.eventTypeRepo.getEventTypeById(eventTypeId);
+    if (event?.teamId) return true;
```

Comment:
Nit: can be 1 line of code since you are evaluating a Boolean to return a Boolean